### DISCUSSION TITLE
Server Error 400 when trying to get forecast GFS data

### DISCUSSION THREAD

**Discussion by josegardo** [pvlib/pvlib-python#1520] [General]
Hi,

I was using forecast GFS data for the last two months without problems, but from yesterday it is not working. This the error:

```
requests.exceptions.HTTPError: Error accessing https://thredds.ucar.edu/thredds/ncss/grid/grib/NCEP/GFS/Global_0p5deg/Best?var=Total_cloud_cover_convective_cloud&var=Total_cloud_cover_entire_atmosphere_Mixed_intervals_Average&var=Temperature_surface&var=Downward_Short-Wave_Radiation_Flux_surface_Mixed_intervals_Average&var=Low_cloud_cover_low_cloud_Mixed_intervals_Average&var=Total_cloud_cover_boundary_layer_cloud_Mixed_intervals_Average&var=Medium_cloud_cover_middle_cloud_Mixed_intervals_Average&var=High_cloud_cover_high_cloud_Mixed_intervals_Average&var=v-component_of_wind_isobaric&var=Wind_speed_gust_surface&var=u-component_of_wind_isobaric&time_start=2022-08-11T00%3A00%3A00%2B00%3A00&time_end=2022-08-12T00%3A00%3A00%2B00%3A00&longitude=-6.136922677556855&latitude=37.37858591008867&vertCoord=100000&accept=netcdf
Server Error (400: NETCDF3 format not supported for ANY_POINT feature type.)
```

Does anybody know the problem?

Thank you!

**Comment by q33420540** [comment]
Yes same question with me😪, I find this question today. Do you got some idea?

**Comment by cwhanse** [comment]
My first guesses are an issue with the server, or a change in server API. Please note that `pvlib/forecast.py` is deprecated and planned for removal, due to not having a maintainer for that capability. See notes [here](https://pvlib-python.readthedocs.io/en/stable/user_guide/forecasts.html).

**Comment by woker001** [comment]
Crash when changing from "pvlib.forecast" to pandas after GFS server API change. So retrieving them one at a time works fine.
